pkg/ddl/notifier/publish_testkit_test.go

```diff
@@ -33,7 +35,7 @@ func TestPublishToTableStore(t *testing.T) {
 CREATE TABLE ddl_notifier (
 	ddl_job_id BIGINT,
 	multi_schema_change_seq BIGINT COMMENT '-1 if the schema change does not belong to a multi-schema change DDL. 0 or positive numbers representing the sub-job index of a multi-schema change DDL',
-	schema_change JSON COMMENT 'SchemaChange at rest',
+	schema_change LONGBLOB COMMENT 'SchemaChangeEvent at rest',
```

Comment:
Because other reviewers and me has no comment, I think you can choose the type as you like. Please `/unhold` after finished.